data/json/construction.json

```diff
@@ -6753,6 +6753,23 @@
     "pre_terrain": "f_hay",
     "post_terrain": "f_archery_target_bale"
   },
+  {
+    "type": "construction",
+    "id": "constr_wooden_flagpole",
+    "group": "build_flagpole",
+    "category": "CONSTRUCT",
+    "time": "30 m",
+    "required_skills": [ [ "fabrication", 0 ] ],
+    "qualities": [ [ { "id": "HAMMER", "level": 2 } ] ],
```

Comment:
probably need some diggging tool too

Reply:
Was not sure how easy to construct to make it. Had digging 2 and drilling at first, but found that too a high barrier. Also had a beam instead of poles first, which justifies that.

Took fence poles as a template. They basically are poles rammed into the ground. Here, we just have two poles nailed together.

But well, a longer pole will need to be deeper in the ground for stability. So will add digging of 1.

Reply:
Done, added digging 1.